Model/Api/ShippingMethods.php

```diff
@@ -319,11 +319,10 @@ public function getShippingMethods($cart, $shipping_address)
             // Load quote from entity id
             $quote = $this->cartHelper->getQuoteById($quoteId);
```

Comment:
In the model class `Model/Api/ShippingMethods.php` exist method `getQuoteById()`.
We can simplify it and just call like `$this->getQuoteById($quoteId)`.

Reply:
done